app/models/hearing.rb

```diff
@@ -187,9 +188,19 @@ def advance_on_docket_motion
       .first
   end
 
+  # returns scheduled datetime object considering the timezones
+  # nil - if hearing_day is nil
+  # datetime in scheduled_in_timezone timezone - if scheduled_datetime and scheduled_in_timezone are present
+  # else datetime in regional office timezone
   def scheduled_for
     return nil unless hearing_day
 
+    # returns datetime in scheduled_in_timezone timezone
+    if scheduled_datetime.present? && scheduled_in_timezone.present?
+      scheduled_datetime_string = scheduled_datetime.strftime("%Y-%m-%d %H:%M:%S %z")
+      return Time.use_zone(scheduled_in_timezone) { Time.zone.parse(scheduled_datetime_string) }
```

Comment:
```suggestion
     return scheduled_datetime.in_time_zone(scheduled_in_timezone)
```

Seems odd to generate a time in string form just to parse it into an object again. I THINK (not 100%, but tests should confirm or deny) that you can coerce the time object into a time zone like this.

Example Reference: https://stackoverflow.com/questions/2695837/rails-convert-utc-datetime-to-another-time-zone

Reply:
I agree with Stephen. The way he has suggested it be implemented is how we have it on our prototype branch. I can confirm that it'll work!